apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx

```diff
@@ -142,50 +177,76 @@ export default function LandingPage({
                 name="myPLTabs"
                 tabs={availableTabs}
                 defaultSelectedTabValue={availableTabs[0].value}
-                onChange={tab => {
-                  // [TODO]: Uncomment this out once
-                  // currentTab affects what content
-                  // is shown.
-                  //setCurrentTab(tab);
-                  console.log(tab);
-                }}
+                onChange={tab => setCurrentTab(tab)}
               />
             </nav>
           )}
         </HeaderBannerNoImage>
       </div>
       <main className={style.wrapper}>
-        {showGettingStartedBanner && RenderGettingStartedBanner()}
-        {lastWorkshopSurveyUrl && RenderLastWorkshopSurveyBanner()}
-        <EnrolledWorkshops />
-        {plCoursesStarted?.length >= 1 && (
-          <section id={'self-paced-pl'}>
-            <Heading2>{i18n.plLandingSelfPacedProgressHeading()}</Heading2>
-            {RenderSelfPacedProgressTable()}
-          </section>
+        {currentTab === 'myPL' && (
+          <>
+            {showGettingStartedBanner && RenderGettingStartedBanner()}
+            {lastWorkshopSurveyUrl && RenderLastWorkshopSurveyBanner()}
+            <EnrolledWorkshops />
+            {plCoursesStarted?.length >= 1 && (
+              <section id={'self-paced-pl'}>
+                <Heading2>{i18n.plLandingSelfPacedProgressHeading()}</Heading2>
+                {RenderSelfPacedProgressTable()}
+              </section>
+            )}
+            {deeperLearningCourseData?.length >= 1 && (
+              <section>
+                <Heading2>Online Professional Learning Courses</Heading2>
+                <ProfessionalLearningCourseProgress
+                  deeperLearningCourseData={deeperLearningCourseData}
+                />
+              </section>
+            )}
+            <section>
+              <Heading2>{i18n.plLandingRecommendedHeading()}</Heading2>
+              {RenderStaticRecommendedPL()}
+            </section>
+          </>
         )}
-        {deeperLearningCourseData?.length >= 1 && (
+        {['myFacilitatorCenter', 'instructors'].includes(currentTab) && (
```

Comment:
The facilitator tab should have more components at some point. While adding those components is outside the scope of this PR, it might be worth separating out this logic

Reply:
Yes for sure, my plan once we started implementing more components was to simplify the logic to a series of `{shouldRenderXComponent && (<XComponent/>)}` where shouldRenderXComponent would contain the logic of checking if the user has the right permissions and any other qualifications